packages/electron-updater/src/BaseUpdater.ts

```diff
@@ -55,12 +57,23 @@ export abstract class BaseUpdater extends AppUpdater {
     this.quitAndInstallCalled = true
 
     try {
-      this._logger.info(`Install: isSilent: ${isSilent}, isForceRunAfter: ${isForceRunAfter}`)
+      let installPathRequiresElevation = false
+      if (process.platform === "win32") {
+        try {
+          const accessTestPath = path.join(path.dirname(process.execPath), "access")
+          fs.writeFileSync(accessTestPath, " ")
+        } catch(err) {
```

Comment:
Couple of notes here.
To avoid odd edge cases for the project that coincidentally includes a file named 'access', could you utilize a random id for this `.tmp` file? Something like this:
```access-${Math.floor(Math.random() * 100)}.tmp```
Also, if the write is successful, I think we should then delete the file afterward